### ISSUE TITLE
New to using tor

### ISSUE THREAD

**Issue by spazlet** [rdeepak2002/reddit-place-script-2022#440]
Hi, I have never used tor before and was wondering what I need to do to run this program with tor.

**Comment by Skyline-9** [comment]
Check out the README for instructions on installing Tor and setting it up. Do you have a specific question about Tor?

**Comment by spazlet** [comment]
> Check out the README for instructions on installing Tor and setting it up. Do you have a specific question about Tor?

Yes, when using the command to run tor, what directory should it be from, because the error I get is "'tor' is not recognized as an internal or external command,
operable program or batch file."

**Comment by Risewill23** [comment]
Check if the current working dir is in the folder with the main.py script also I would try to quickly do it because I think the event is ending in 10 minutes
